python/cuspatial/cuspatial/_lib/interpolate.pxd

```diff
@@ -4,21 +4,27 @@
 # cython: language_level = 3
 # distutils: include_dirs = cuspatial/bindings/
 
-from cudf._libxx.table cimport *
+from cudf._libxx.table cimport table, \
+                               column_view, \
```

Comment:
can avoid `\` if you surround imports in `( ... )`.

Reply:
yes, thanks